src/a2m.cpp

```diff
@@ -199,7 +200,7 @@ bool Ca2mLoader::load(const std::string &filename, const CFileProvider &fp)
     orgptr = org + alength;
   }
 
-  if (orgptr - org < needed) {
+  if (orgptr - org < (ssize_t)needed) {
```

Comment:
pointer - pointer   =>  signed long
_needed_ in most locations are used as an unsigned number except this single comparison

Reply:
From MSVC: `error C2065: 'ssize_t': undeclared identifier`